### ISSUE TITLE
generate with pre-trained models gives gibberish data

### ISSUE THREAD

**Issue by lpanchma** [NVIDIA/sentiment-discovery#50]
When I try to run generate on the pre-trained models included in the repo, I get gibberish output

python3 /sentiment-discovery/generate.py --gen_length 100 --load_model /SDModels/imdb_clf_16.pt
Creating mlstm
/sentiment-discovery/generate.py:112: UserWarning: invalid index of a 0-dim tensor. This will be an error in PyTorch 0.5. Use tensor.item() to convert a 0-dim tensor to a Python number
  neuron = neuron[0]

ªS¦.ßý;Ù·XÜ¿}íø{ÁU=0é)ò$EEÙ¢N'j$¼ûeä²0Ã(C#¤±ßÿ[×fnxXÔ/s©

I have torch
>>> import torch
>>> print (torch.__version__)
0.4.1
>>>

**Comment by raulpuric** [comment]
As I've mentioned on other issues in this repo, we do not provide anymore support for generate.py unfortunately.
